### ISSUE TITLE
useUnknownInCatchVariables reset

### ISSUE THREAD

**Issue by gulderov** [opensearch-project/oui#887]
Please reset `useUnknownInCatchVariables` to its default value to restore common behavior. 

Note that the `useUnknownInCatchVariables` was disabled in #879

The proposed change:
* Remove `useUnknownInCatchVariables` from tsconfig file

**Comment by joshuarrrr** [comment]
Thanks @gulderov - are you planning to do this in a follow-up PR to #879?

**Comment by gulderov** [comment]
The necessary changes have already been implemented in #879. Once that pull request is merged, this issue will be resolved and can be closed
